src/simulations/paramscan.jl

```diff
@@ -25,13 +25,19 @@ see the example below.
 The following keywords modify the `paramscan` function:
 
 - `include_constants::Bool=false` determines whether constant parameters should be
-  included in the output `DataFrame`.
+  included in the output `DataFrame`. If false, the varying parameters will still be
+  included in the output.
```

Comment:
I don't understand this. If `false` it is not included...?

Reply:
Yeah it is a bit confusing, if false then only the varying parameters will be included, if true than the constant parameters will be included as well. I have updated this.